timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -580,13 +580,18 @@ def _train_multi(
                     )
                 logger.info(fit_log_message)
 
-                with tqdm.external_write_mode():
-                    model_names_trained += self.tune_model_hyperparameters(
-                        model,
-                        time_limit=time_left,
-                        train_data=train_data,
-                        val_data=val_data,
-                        hyperparameter_tune_kwargs=hyperparameter_tune_kwargs,
+                if contains_searchspace(model._user_params):
```

Comment:
judging by `_user_params` we should not be accessing it from the outside. If there isn't a public method, should we create one?

Reply:
It seems that there is no public method for accessing the model parameters before initialization, so I created one